Documentation/ceph-csi-snapshot.md

```diff
@@ -26,6 +22,10 @@ In short, as the documentation describes it:
 > VolumeSnapshotClass provides a way to describe the “classes” of storage when
 > provisioning a volume snapshot.
 
+## Upgrade Snapshot API
+
+If your Kubernetes Version supports newer v1beta1/v1 snapshot API, follow the upgrade guide [here](https://github.com/kubernetes-csi/external-snapshotter/tree/v4.0.0#upgrade).
```

Comment:
I don't understand, when do they need to follow this upgrade guide? When converting from v1beta1 to v1?

Reply:
Both when upgrading from alpha to beta and beta to v1.

Reply:
Ok thanks for updating that statement to clarify